src/main/java/hello/HelloWorldHttpController.java

```diff
@@ -16,6 +16,8 @@ public String greeting(@RequestParam(value="name", required=false, defaultValue=
         if (true){
             int r = 4 + 5;
             System.out.print(r);
+            String token2 = new Application().generateCSRFToken();
+            System.out.print(token2);
         }
         else {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this "else" on the same line that the previous closing curly brace. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.37.128.36:9000/coding_rules#rule_key=squid%3ARightCurlyBraceSameLineAsNextBlockCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.37.128.36:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)